Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -11,16 +11,18 @@ private let log = Logger.browserLogger
 class ShareExtensionHelper: NSObject {
     fileprivate weak var selectedTab: Tab?
 
-    fileprivate let selectedURL: URL
+    fileprivate let selectedURL: URL // for non copy ones
+    fileprivate let fileURL: URL? // for copy type
     fileprivate var onePasswordExtensionItem: NSExtensionItem!
     fileprivate let browserFillIdentifier = "org.appextension.fill-browser-action"
 
-    init(url: URL, tab: Tab?) {
+    init(url: URL, file: URL?, tab: Tab?) {
```

Comment:
setting a default value for file `file: URL? = nil` will let us avoid having to add `file:nil` when creating it